examples/standalone_malcontent_example.py

```diff
@@ -0,0 +1,42 @@
+import os
+import tempfile
+
+from rich import print
+
+import acuvity
+from acuvity import Acuvity
+
+# Initialize Acuvity
+s = Acuvity(
+    security=acuvity.Security(
+        token=os.getenv("ACUVITY_TOKEN", ""),
+    )
+)
+
+# Create temporary file with the content
+def create_temp_file(content):
+    with tempfile.NamedTemporaryFile(mode='w+', delete=False) as temp:
+        temp.write(content)
+        return temp.name
+
+# PII Guard config content
+pii_config_content = """guardrails:
+  - name: toxic"""
```

Comment:
for these examples here, I'm also not entirely understanding why we would be writing out the guard config. If you want to keep everything inline in code, then one would just pass this as a dict or the proper guard config type to scan below

Reply:
you can do it either way. I have try to keep standalone files in this format but, will change the names it should not be pii.

Reply:
My thinking was all standalone prefixed files will have the same way of doing the guard config. When we were initially doing standalone we wanted to show a few examples on how to do the config through file reading. 

Thats the only reason I kept it. But, what you are saying is also fair.